src/Symfony/Component/Serializer/NameConverter/NameConverterInterface.php

```diff
@@ -21,16 +21,20 @@
     /**
      * Converts a property name to its normalized value.
      *
-     * @param string $propertyName
+     * @param string|object $class
+     * @param string        $propertyName
+     *
      * @return string
      */
-    public function normalize($propertyName);
+    public function normalize($class, $propertyName);
```

Comment:
@weaverryan even if you make it optional, it is still a BC break: people implementing the interface will have to change their code, otherwise they will get PHP errors about signature mismatches.